data/filedefs/filetypes.nsis

```diff
@@ -23,8 +23,8 @@ commentbox=comment
+variables=${nsisdir} $0 $1 $2 $3 $4 $5 $6 $7 $8 $9 $r0 $r1 $r2 $r3 $r4 $r5 $r6 $r7 $r8 $r9 $\n $\r $\t $$ $admintools $appdata $cdburn_area $cmdline $commonfiles $commonfiles32 $commonfiles64 $cookies $desktop $documents $exedir $exefile $exepath $favorites $fonts $history $hwndparent $instdir $internet_cache $language $localappdata $music $nethood $outdir $pictures $pluginsdir $printhood $profile $programfiles $programfiles32 $programfiles64 $quicklaunch $recent $resources $resources_localized $sendto $smprograms $smstartup $startmenu $sysdir $temp $templates $videos $windir $_click $_outdir
```

Comment:
I see this removed `` $` ``, `$'`, `$"` and `$0,`, is it on purpose?

Reply:
Yes, they are removed on purpose because they are not listed in the [documentation](https://nsis.sourceforge.io/Docs/Chapter4.html#varuser).

I assume they were already wrong for the last decade :).